packages/ns-api/files/ns.update

```diff
@@ -89,19 +88,30 @@ def install_package_updates():
 def check_system_update():
     e_uci = EUci()
     current_version = get_system_version()
-    ret = {"currentVersion": f'NethSecurity {current_version}', "lastVersion": "", "scheduledAt": get_update_schedule()}
+    data = {"currentVersion": f'NethSecurity {current_version}', "lastVersion": "", "scheduledAt": get_update_schedule()}
+    url = e_uci.get('ns-plug', 'config', 'repository_url')
```

Comment:
This could raise an exception if repository_url is not set. Please make sure to catch the exception as before.